### ISSUE TITLE
Leveler will not work entire width of silage bunker in Leveling/Compacting mode

### ISSUE THREAD

**Issue by StoneC0ld** [Courseplay/courseplay#2196]
I created my course exactly as per several tutorials on Youtube, and configured the setting exactly per the tutorials.  However, very rarely will it ever actually work the full width of the bunker.  

I just tried about 30 times in a row, driving close to the waiting point (where I've been told I have the best chances of actually making this work), starting the driver, and watching it insist on going straight down the middle, stop the driver and drive back to the waiting point again.  My last round of grass cutting, I let it go, and it was just going back and forth and back and forth straight down the middle, not doing anything on the sides.  

I'm working at the BGA on Rattlesnake Valley V2, using the narrower of the two in-game levelers on the back of the snake-skinned tractor with the tracks (Field Viper?  Not sure the name of the tractor off the top of my head), but the choice in tractor does not seem to matter.

Not sure what information might be needed, but I can try to provide information as needed/requested.

**Comment by pops64** [comment]
Did you adjust the blade width setting? There may be silage underneath the ground that it is detecting. Try setting it up in GCV or Snoska.  If not we need a picture of what your course looks like. And a log with debug channel 10

**Comment by StoneC0ld** [comment]
Blade width is set exactly the same as Mister Moose Gaming at 5.0, who uses the same leveler in his tutorial.  I can try it on one of the default maps, but it's probably going to take at least until tomorrow before I can take the time to do that test.

[log.txt](https://github.com/Courseplay/courseplay/files/1171533/log.txt)
![fs17 courseplay problem](https://user-images.githubusercontent.com/16530220/28549633-003542d8-70a1-11e7-8d58-5584ecada152.jpg)

I also recorded the session that produced this log.txt, and can upload to Youtube and paste the link here.

**Comment by pops64** [comment]
The problem may be the very little bit of sillage that is in front of the bunker. It probably sees it but can't do anything about it and keeps trying.

**Comment by StoneC0ld** [comment]
Tried manually clearing up the bits of chaff in front of the bunker by pushing it into the bunker.  Continues to only run straight down the middle.

https://www.youtube.com/watch?v=UA4nFCE__rE
This is the recorded video of the session that produced the log.txt in my previous post, which shows the on-screen numbers that the screenshot didn't get because courseplay wasn't activated at the moment I took the screenshot.
Attached here is the log.txt from my test after clearing the area in front of the bunker.  I do have a partial recording of this test, but neglected to start the recording until a minute or so before ending the test.
[log.txt](https://github.com/Courseplay/courseplay/files/1173233/log.txt)
